public/main.js

```diff
@@ -0,0 +1,170 @@
+// @ts-check
+
+// const root = document.querySelector('ul');
+// function displayMessage(type, message = '') {
+//   const li = root.appendChild(document.createElement('li'))
+//   li.textContent = `[${type}] ${message}`
+// }
+const connectDiv = document.querySelector('#connect')
+const [connectAsAssistantBtn, connectAsTraineeBtn, disconnectBtn] = Array.from(connectDiv.querySelectorAll('button'))
+const role = document.querySelector('#role')
+const actions = document.querySelector('#actions');
+function updateActions(...children) {
+  actions.replaceChildren(...children)
+}
+
```

Comment:
This file serves tests purposes currently.
And since it is a first draft, I did not focus on code style, for that we can install ESLint (and even prettier) later